draft-ietf-acme-acme.md

```diff
@@ -1394,15 +1395,10 @@ client fails to complete the required actions before the "expires" time, then
 the server SHOULD change the status of the order to "invalid" and MAY
 delete the order resource.
 
-The server MUST begin the issuance process for the requested certificate and
-update the order resource with a URL for the certificate once the client has
-fulfilled the server's requirements.   If the client has already satisfied the
-server's requirements at the time of this request (e.g., by obtaining
-authorization for all of the identifiers in the certificate in previous
-transactions), then the server MUST proactively issue the requested certificate
-and provide a URL for it in the "certificate" field of the order.  The server
-MUST, however, still list the completed authorizations in the "authorizations"
-array.
+Once the client has fulfilled the server's requirements, the server MUST update
+the order resoruce with a URL for the certificate.  It SHOULD begin the issuance
+process at this point, but MAY postpone issuance until it receives a GET request
+for the certificate URL.
```

Comment:
Let's Encrypt currently constructs certificate URLs using the serial # of the certificate. It's not a big deal but I wanted to note that a MUST for updating the order with a certificate URL when the order is authorized and a MAY for postponing issuance until GET means CAs that delay issuance can't reference any certificate attributes in the URL since they have to construct it before issuance.